docs/network-flow-visibility.md

```diff
@@ -283,7 +287,46 @@ it is deployed following the [deployment steps](#deployment-steps-1), the
 ClickHouse server is already exposed via a K8s Service, and no further
 configuration is required. If a different FQDN or IP is desired, please use
 the URL for `clickHouse.databaseURL` in the following format:
-`tcp://<ClickHouse server FQDN or IP>:<ClickHouse TCP port>`.
+`<Protocol>://<ClickHouse server FQDN or IP>:<ClickHouse port>`.
+
+#### Support secure connections to ClickHouse database
+
+Since Antrea v1.13, you can enable TLS when connecting to ClickHouse Server by
+setting `clickHouse.databaseURL` with protocol `tls` or `https`. You can also
+change the value of `clickHouse.tls.insecureSkipVerify` to determine whether
+to skip the verification of the server's certificate. If you want to provide
+a custom CA certificate, you can set `clickHouse.tls.customizedCACert` to
```

Comment:
the name is `customCACert` and not `customizedCACert`; but as per my comment above it should probably be `caCert`